.devcontainer/devcontainer.json

```diff
@@ -6,7 +6,9 @@
 	"image": "mcr.microsoft.com/devcontainers/base:jammy",
 	"features": {
 		"ghcr.io/devcontainers/features/dotnet:2": {},
-		"ghcr.io/devcontainers/features/node:1": {}
+		"ghcr.io/devcontainers/features/node:1": {},
+		"ghcr.io/devcontainers/features/python": "3.12",
+		"ghcr.io/devcontainers/features/azure-cli": "latest"
```

Comment:
Tip: If you havent already, you could enable Codespace pre-builds for this repo, given how many features you have, thatd make it faster to start up.
Not a big deal though since this repo isn't as likely to be used as a Codespace.

Reply:
Done now :)